mautic/mautic
# Fix primary email company tokens

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:
This PR fix wrong primary company tokens in email


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create company Test1 with address 1  Test1
3. Create company and assign company Test1
4. Create company Test2 with address 1 Test2
5. Then assign company Test2 to contact as primary but let also Test1 as secondary company
6. Create email with tokens
7. 
```
{contactfield=email}
{contactfield=companyname}
{contactfield=companyaddress1}
```

8. You should receive companyname and address1 from secondary company. Expect primary company data
9. After PR should work




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10733"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
